cnxml/cli.py

```diff
@@ -50,3 +53,19 @@ def collxml(argv=None):
 
     retcode = errors and 1 or 0
     return retcode
+
+
+def extract_metadata(argv=None):
+    args = _arg_parser().parse_args(argv)
+
+    files = [Path(x) for x in args.xml]
+
+    while files:
+        file = files.pop(0)
+        with file.open('rb') as fb:
+            xml = etree.parse(fb)
+        metadata = parse_metadata(xml)
+        print(json.dumps(metadata, sort_keys=True, indent=2))
+        if files:
+            print('-' * 10)
```

Comment:
You don't want to return a json list if there are multiple files?

Reply:
I wasn't going to get too involved, but we could. 👍